css-two-column-layout/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <header>
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>
+          We've dedicated thousands of man hours to making healthcare institutions in the Heluim network run 10x more efficiently and deliver an unparalleled quality of care.
+        </p>
+      </header>
+    </div>
+    <div class="row">
+      <div class="column">
```

Comment:
You want to make sure that the columns have an explicit width set on them. While this structure is mostly working, its relying on flexbox setting the size of the columns, which is something you want to avoid.